### PR TITLE
Update fieldset styling

### LINKED ISSUE #7428: Remove background from fieldset

### PR DESCRIPTION
**Pull request by falko100**
… legend tag for better visibility.

As requested in issue #7428 

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

**Comment by falko100** [comment]
Before:
image
image


After:
image
image

**Comment by zepfietje** [comment]
Since the fieldset can be used on a variety of backgrounds (e.g. wrapped in a section or not), we can't just apply a background color to the label.

Could you refactor this use a border instead of ring, remove the backgrounds and shadow, @falko100?

Only downside is that the border will be inside the element instead of outside like the ring, but there is no other way to return the default fieldset behavior where the border doesn't cut through the label.

Mark as ready for review when done. 🙂

**Comment by falko100** [comment]
New version:
image
image

File: packages/support/resources/views/components/fieldset.blade.php [hunk 1]
```diff
@@ -5,12 +5,12 @@
 <fieldset
     {{
         $attributes->class([
-            'fi-fieldset rounded-xl bg-white p-6 shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10',
+            'fi-fieldset rounded-xl p-6 shadow-sm border border-gray-950/5 dark:border-white/10',
```
Comment (zepfietje) [inline packages/support/resources/views/components/fieldset.blade.php]:
```suggestion
            'fi-fieldset rounded-xl p-6 shadow-sm border border-gray-200 dark:border-white/10',
```

File: packages/support/resources/views/components/fieldset.blade.php [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
     {{
         $attributes->class([
-            'fi-fieldset rounded-xl bg-white p-6 shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10',
+            'fi-fieldset rounded-xl p-6 shadow-sm border border-gray-200 dark:border-white/10',
         ])
     }}
 >
     @if (filled($label))
-        <legend class="text-sm font-medium leading-6">
+        <legend class="text-sm font-medium leading-6 px-1 -mx-1">
```
Comment (zepfietje) [inline packages/support/resources/views/components/fieldset.blade.php]: These changes are no longer needed, are they?

Comment (danharrin) [reply packages/support/resources/views/components/fieldset.blade.php]: We used `-ml-2 px-2` in v2 so restored that

Comment (falko100) [reply packages/support/resources/views/components/fieldset.blade.php]: It adds a little extra spacing around the legend

Comment (zepfietje) [reply packages/support/resources/views/components/fieldset.blade.php]: Okay I'll pull in the changes and properly review later.

Comment (falko100) [reply packages/support/resources/views/components/fieldset.blade.php]: Anything I can do for this?

Comment (zepfietje) [reply packages/support/resources/views/components/fieldset.blade.php]: No action required on your end. 🙂

**Comment by zepfietje** [comment]
Thanks guys!

**Pull request merged**